keep/api/utils/tenant_utils.py

```diff
@@ -63,13 +199,87 @@ def create_api_key(
         role=role.get_name(),
     )
     session.add(new_installation_api_key)
+
     if commit:
         session.commit()
+
     logger.info(
         "Created API key",
         extra={"tenant_id": tenant_id, "unique_api_key_id": unique_api_key_id},
     )
-    return api_key
+
+    if is_system:
```

Comment:
why?..

Reply:
So this I added because when called by get_or_create_api_key is_system will be true. If its not, then its being called by the POST "/apikey" endpoint.

These have different needs. get_or_create_api_key expects a string which should be the api_key secret. To be able to optimistically update the UI,  the POST api key endpoint returns the full object and the secret (TenantApiKey & {"secret": api_key}).